package.json

```diff
@@ -7,11 +7,14 @@
     "dependencies": {
         "@types/node": "^10.14.14",
         "file-saver": "^1.3.8",
-        "sparser": "^1.4.4"
+        "sparser": "^1.4.4",
+        "typescript": "^3.5.3"
     },
     "description": "A language aware diff tool.",
     "devDependencies": {
+        "@types/node": "^10.14.13",
         "ace-builds": "^1.4.5",
+        "typescript": "^3.5.2",
```

Comment:
What is the benefit for adding TypeScript both as a regular dependency and a dev dependency?